Exiled.Events/Patches/Generic/Scp173BeingLooked.cs

```diff
@@ -36,63 +172,42 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
             Label cnt = generator.DefineLabel();
             Label isTutorial = generator.DefineLabel();
 
-            int addCheckOffset = 4;
-            int addCheck = newInstructions.FindLastIndex(instruction => instruction.Calls(Method(typeof(Physics), nameof(Physics.Linecast), new[] { typeof(Vector3), typeof(Vector3), typeof(int) }))) + addCheckOffset;
-
-            LocalBuilder player = generator.DeclareLocal(typeof(Player));
-
+            LocalBuilder currentPlayer = generator.DeclareLocal(typeof(Player));
+            LocalBuilder scp173Player = generator.DeclareLocal(typeof(Player));
             LocalBuilder turnedPlayers = generator.DeclareLocal(typeof(HashSet<Player>));
 
-            newInstructions.InsertRange(addCheck, new CodeInstruction[]
-            {
-                // Player.get(current player in all hub)
-                new(OpCodes.Ldloc_3),
-                new(OpCodes.Call, Method(typeof(Player), nameof(Player.Get), new[] { typeof(ReferenceHub) })),
-                new(OpCodes.Dup),
-                new(OpCodes.Stloc, player.LocalIndex),
-
-                // Player is null (Generic static ref hub for example)
-                new(OpCodes.Brfalse_S, jne),
+            int removeTurnedPeanutOffset = 2;
+            int removeTurnedPeanut = newInstructions.FindIndex(instruction => instruction.Calls(PropertyGetter(typeof(HashSet<ReferenceHub>), nameof(HashSet<ReferenceHub>.Count)))) + removeTurnedPeanutOffset;
 
-                // Player.role == Tutorial
-                new(OpCodes.Ldloc, player.LocalIndex),
-                new(OpCodes.Callvirt, PropertyGetter(typeof(Player), nameof(Player.Role))),
-                new(OpCodes.Callvirt, PropertyGetter(typeof(API.Features.Roles.Role), nameof(API.Features.Roles.Role.Type))),
-                new(OpCodes.Ldc_I4_S, (int)RoleType.Tutorial),
-
-                // Skip to checking if tutorial should be blocked
-                new(OpCodes.Beq_S, isTutorial),
-
-                // Scp173.TurnedPlayers.Remove(Current 173)
-                new(OpCodes.Call, PropertyGetter(typeof(API.Features.Scp173), nameof(API.Features.Scp173.TurnedPlayers))),
-                new(OpCodes.Stloc, turnedPlayers.LocalIndex),
-                new(OpCodes.Ldloc, turnedPlayers.LocalIndex),
+            newInstructions.InsertRange(removeTurnedPeanut, new CodeInstruction[]
```

Comment:
You can write this without `scp173Player` variable, something like:
```cs
new(OpCodes.Call, PropertyGetter(typeof(API.Features.Scp173), nameof(API.Features.Scp173.TurnedPlayers))),
new(OpCodes.Ldarg_0),
new(OpCodes.Ldfld, Field(typeof(PlayableScp), nameof(PlayableScp.Hub))),
new(OpCodes.Callvirt, Method(typeof(Player), nameof(Player.Get), new[] { typeof(ReferenceHub) })),
new(OpCodes.Callvirt, Method(typeof(HashSet<Player>), nameof(HashSet<Player>.Remove))),
new(OpCodes.Pop),
```

Reply:
Minor optimization but sure, that works